src/operation.ts

```diff
@@ -419,13 +419,33 @@ export default class Operation {
   }
 
   /**
-   * Determine if the operation has a request body.
+   * Determine if the operation has any request bodies.
    *
    */
   hasRequestBody(): boolean {
     return !!this.schema.requestBody;
   }
 
+  /**
+   * Retrieve a specific request body content schema off this operation.
+   *
+   * @see {@link https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#mediaTypeObject}
+   * @see {@link https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.1.0.md#mediaTypeObject}
+   * @param mediaType Specific request body media type to retrieve if present.
+   */
+  getRequestBody(mediaType: string) {
+    if (!this.hasRequestBody()) {
+      return false;
+    }
+
+    const requestBody = this.schema.requestBody;
+    if (RMOAS.isRef(requestBody) || !(mediaType in requestBody.content)) {
```

Comment:
Why return false if it's a ref? It's probably worth leaving those details as a comment here. I also don't see any tests for when the req body is a ref.